Nos.xcodeproj/project.pbxproj

```diff
@@ -2034,11 +2046,11 @@
 /* Begin PBXTargetDependency section */
 		3AD3185D2B294E9000026B07 /* PBXTargetDependency */ = {
 			isa = PBXTargetDependency;
-			productRef = 3AD3185C2B294E9000026B07 /* plugin:XCStringsToolPlugin */;
+			productRef = 3AD3185C2B294E9000026B07 /* XCStringsToolPlugin */;
 		};
 		3AEABEF32B2BF806001BC933 /* PBXTargetDependency */ = {
 			isa = PBXTargetDependency;
-			productRef = 3AEABEF22B2BF806001BC933 /* plugin:XCStringsToolPlugin */;
```

Comment:
Not related to this PR. But this look weird, I'm seeing those in my PRs as well. It might worth checking this when we start tackling the issue of switching between branches, I have no idea, but it looks like it could be disrupting the switft package manager.